src/components/AddressInfo/index.tsx

```diff
@@ -39,10 +39,10 @@ const StyledBlock = styled(Block)`
 interface Props {
   safeName?: string
   safeAddress: string
-  ethBalance?: string
+  ethBalance?: number | string
```

Comment:
can we maybe unify the type? e.g make sure that ethBalance is always a string

Reply:
I thought about that, but I decided to just accept both types and let that change for future changes. Maybe adding a TODO will help ups keep track of that?

Reply:
mh... I ended up by removing all the `number` occurrences, it looks like it's working fine